src/uct/api/uct_def.h

```
@@ -222,8 +222,12 @@ typedef ucs_status_t (*uct_pending_callback_t)(uct_pending_req_t *self);
  *                       operations on this @a ep will fail with the error code
  *                       passed in @a status.
  * @param [in]  status   Status indicating error.
+ *
+ * @return @ref UCS_OK   - The error was handled successfully.
+ *         Otherwise     - Pass error code back to transport layer.
```

Otherwise - the error was not handled and is returned back to the transport